Source/Core/Core/ConfigManager.cpp

```diff
@@ -728,12 +729,14 @@ void SConfig::SetRunningGameMetadata(const std::string& game_id, u64 title_id, u
   if (game_id == "00000000")
   {
     m_title_description.clear();
+    Discord::UpdateDiscordPresence(m_title_description);
```

Comment:
UI Common shouldn't be within Core. When I meant that the code shouldn't be within Core and handled by the front-end, I mean the front-end calls this when booting the game, not the Core. Core should have *no* direct usage of anything from the UI libraries.